source/packages/services/installer/src/models/answers.ts

```diff
@@ -194,28 +194,46 @@ export interface CertificateActivator extends ServiceModuleAttributes {
 }
 
 export interface CertificateVendor extends ServiceModuleAttributes {
-    commandsFunctionName?: string;
-    assetLibraryFunctionName?: string;
-    providingCSRs?: boolean;
-    caCertificateId?: string;
-    useDefaultPolicy?: boolean;
-    rotatedCertificatePolicy?: string;
-    // Application Configuration
-    certificatesPrefix?: string;
-    certificatesSuffix?: string;
-    presignedUrlExpiryInSeconds?: number;
-    rotateCertificatesThingGroup?: string;
-    getSuccessTopic?: string;
-    getFailureTopic?: string;
-    getRootTopic?: string;
-    ackSuccessTopic?: string;
-    ackFailureTopic?: string;
-    ackRootTopic?: string;
-    deletePreviousCertificate?: boolean;
-    deviceStatusSuccessKey?: string;
-    deviceStatusSuccessValue?: string;
-    certificateExpiryInDays?: number;
-    registryMode?: 'AssetLibrary' | 'DeviceRegistry';
+  commandsFunctionName?: string;
+  assetLibraryFunctionName?: string;
+  providingCSRs?: boolean;
+  acmpcaEnabled?: boolean;
+  caCertificateId?: string;
+  //acmpca related configurations
+  caArnAcmpca?: string;
+  acmpcaSingnalingAlgorithm?: string;
+  acmpcaCrossAccountRoleArn?: string;
+  acmpcaRegion?: string;
+  setPcaAliases?: boolean;
+  pcaAliases?: CAAliases;
+  pcaAlias?: string;
+  pcaArn?:string;
+  pcaFinished?: boolean;
+  setIotCaAliases?: boolean;
+  iotCaAliases?: CAAliases;
+  iotCaAlias?: string;
+  iotCaID?:string;
+  iotCaFinished?: boolean;
+  //
+  useDefaultPolicy?: boolean;
+  rotatedCertificatePolicy?: string;
+  // Application Configuration
+  certificatesPrefix?: string;
+  certificatesSuffix?: string;
+  presignedUrlExpiryInSeconds?: number;
+  rotateCertificatesThingGroup?: string;
+  getSuccessTopic?: string;
+  getFailureTopic?: string;
+  getRootTopic?: string;
+  ackSuccessTopic?: string;
+  ackFailureTopic?: string;
+  ackRootTopic?: string;
+  deletePreviousCertificate?: boolean;
+  deviceStatusSuccessKey?: string;
+  deviceStatusSuccessValue?: string;
+  certificateExpiryInDays?: number;
+  registryMode?: "AssetLibrary" | "DeviceRegistry";
+
```

Comment:
nit: unnecessary extra line
(note: we've standardized on formatting, and if using VSCode, these files can be re-saved and the formatting should be applied automatically)

Reply:
will fix